data/modules/TradeShips/Events.lua

```diff
@@ -159,26 +142,31 @@ Event.Register("onShipDocked", onShipDocked)
 
 local onShipUndocked = function (ship, starport)
 	if Core.ships[ship] == nil then return end
+	local trader = Core.ships[ship]
+	
+	if trader.starport.type == "STARPORT_ORBITAL" then
+		-- fly to the limit of the starport frame
+		ship:AIFlyTo(starport)
```

Comment:
It’s still unclear, is there any point in this stage?

Reply:
There's a high chance of ships crashing into walls if given the command to fly to star directly when they are inside station. Let this be for now. This command will still be given, when `AIFlyTo` get completed (~15km from station).

Reply:
It would be good to see confirmation of this statement, as I already wrote, this does not reproduce for me - you have the same chance of leaving the orbital no matter what autopilot command you run.

Reply:
There is not much to add, besides what I have already described in this [comment](https://github.com/pioneerspacesim/pioneer/pull/5647#issuecomment-1763326176). Try to fly to the center of `Jobes Pad` (area between landings pads) and give ship, for example, command to fly to current station. There's almost a guarantee ship will not fly to the exit, but will fly upwards to the wall. Or just give this command, when exit is behind ship and it's moving from it. The higher the speed of the ship, the higher the probability that it will simply fly to the wall, not upwards. 
These two behaviors will occur even if there is no obstructor on the way out. I can reproduce this on current release, even on current master.

Reply:
Please explain why do you give the `AIFlyTo(starport)` command if
> ... give ship, for example, command to fly to current station. There's almost a guarantee ship will not fly to the exit, but will fly upwards to the wall.

Reply:
Decided to try giving command to fly to low orbit of first system's star immediatly when ship undocks once more and I haven't seen much difference in how ship behaves from previous choice, so I pushed change so now ship will do exactly this. When I first tested this, it seemed to me that with this choice everything just gotten worse, so I decided to quickly revert to `AIFlyTo`.
At this point, let's just consider this issues unrelated to `TradeShips` code. Separate issues can be opened for this if needed.